sql/core/src/test/scala/org/apache/spark/sql/connector/V1ReadFallbackSuite.scala

```diff
@@ -129,8 +127,8 @@ class TableWithV1ReadFallback(override val name: String) extends Table with Supp
 
   override def schema(): StructType = V1ReadFallbackCatalog.schema
 
-  override def capabilities(): util.Set[TableCapability] = {
-    util.EnumSet.of(TableCapability.BATCH_READ)
+  override def capabilities(): java.util.Set[TableCapability] = {
+    java.util.EnumSet.of(TableCapability.BATCH_READ)
```

Comment:
hmm, why do we touch these unrelated test suites?

Reply:
compile error. cannot find `Set` in `org.apache.spark.sql.connector.util`.